pkgs/development/libraries/amdvlk/default.nix

```diff
@@ -21,13 +21,13 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "amdvlk";
-  version = "2020.Q4.6";
+  version = "2021.Q1.2";
 
   src = fetchRepoProject {
     name = "${pname}-src";
     manifest = "https://github.com/GPUOpen-Drivers/AMDVLK.git";
     rev = "refs/tags/v-${version}";
```

Comment:
```suggestion
  src = fetchFromnGitHub {
    owner = "GPUOpen-Drivers";
    repo = "AMDVLK";
    rev = "v-${version}";
```

Reply:
That doesn't fetch much useful, see:

https://github.com/GPUOpen-Drivers/AMDVLK

This uses repo tools.